pinot-common/src/main/java/com/linkedin/pinot/startree/hll/HllConfig.java

```diff
@@ -30,8 +31,11 @@
  * segment builder will generate corresponding hll derived fields on the fly.
  */
 public class HllConfig {
```

Comment:
To move this to table config, this class should play nicely with Jackson. **Add tests** to verify conversion to json and back works correctly. 
Fields like hllFieldSize are derived fields. Those will not come from json but should be set correctly. Missing fields can throw errors.

Reply:
good catch, checked this